medusa/tv/series.py

```diff
@@ -548,6 +549,11 @@ def release_required_words(self):
         """Return release ignore words."""
         return [v for v in (self.rls_require_words or '').split(',') if v]
 
+    @staticmethod
+    def normalize_status(series_status):
+        """Return a normalized status given current indexer status."""
+        return STATUS_MAP.get(series_status.lower()) if series_status else 'Unknown'
```

Comment:
This will return None if the status is not in the map. Are you sure this is wanted?